piano-hero/mobile/src/main/java/com/novoda/pianohero/SimpleNotesOutputView.java

```diff
@@ -4,14 +4,11 @@
 import android.support.annotation.Nullable;
 import android.util.AttributeSet;
 import android.view.View;
+import android.view.inputmethod.InputMethodManager;
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import java.util.Locale;
-
-public class SimpleNotesOutputView extends LinearLayout {
-
-    private final SimplePitchNotationFormatter simplePitchNotationFormatter = new SimplePitchNotationFormatter();
+public class SimpleNotesOutputView extends LinearLayout implements GameMvp.View {
```

Comment:
a separate java class that implements GameMvp.View might work. That class could take this AndroidView as a collaborator, but then could perform the other things on its own. E.g. the showKeyboard/Input, doesn't need to be part of this AndroidView.